test/components/previews/selection_preview/default.html.erb

```diff
@@ -0,0 +1,17 @@
+<div data-controller="selection">
+  <% (0...3).each do |index| %>
+    <div>
+      <input
```

Comment:
**nitpick (blocking):** I know we do not have a component for this, but can you please keep it in line with the Flowbite style: https://flowbite.com/docs/components/forms/#checkbox

Reply:
I thought the selection controller should be the :star: of the preview. Please see e019021985ff32e2b8378ecd2a0fa724db063c2c.

Reply:
Thanks, much better